keys/key_client.go

```diff
@@ -15,21 +15,48 @@
 package keys
 
 import (
+	"bytes"
 	"encoding/hex"
 	"fmt"
 
+	acm "github.com/hyperledger/burrow/account"
```

Comment:
I'm not a fan of it either particularly, though at least it's short. I used it because there was a pre-existing usage of it in a few places. Based on the fact previous authors use `dbm` for `go-db`, package `db` I sort of read it as 'database manager' and 'account manager'.

My two reasons for using an alias were:
1. To leave 'account' available for actual accounts - actually this doesn't happen a lot because there are usually `<something>Account`s.
2. Because I don't like `account.Account` as a type name, yet nothing came immediately to mind to replace it.

I don't think these are particularly strong reasons though I'd be tempted to pick another name for the `account` package if I did drop the alias. `state` perhaps, but a bit generic...

Reply:
FYI I'm going to do something about this, but in future single-purpose PR since it will touch almost every file